app/scripts/configs/tracks-info.js

```diff
@@ -1812,6 +1815,7 @@ export const TRACKS_INFO = [
       labelTopMargin: 0,
       labelBottomMargin: 0,
       minHeight: 20,
+      maxAnnotationHeight: 'none',
```

Comment:
In some of my recent pull requests I have added track options, with `null` default values. Is using a "none" string preferred?

Reply:
No, I think you're right and null is actually better. Saves an extra if clause. The only time an explicit 'none' or something like that might be better is where 0 is a valid value.